### ISSUE TITLE
Incompatible with oauth2client 2.0?

### ISSUE THREAD

**Issue by logandg** [google/google-api-python-client#179]
I'm trying out this library for the first time and ran into an error using the build() function.

```
In [1]: from googleapiclient.discovery import build
                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
In [2]: service = build('youtube', 'v3') 

Traceback (most recent call last):                                                                                                                                                                                                                               
  File "/usr/local/lib/python3.5/site-packages/googleapiclient/discovery_cache/__init__.py", line 38, in autodetect                                                                                                                                              
    from . import file_cache                                                                                                                                                                                                                                     
  File "/usr/local/lib/python3.5/site-packages/googleapiclient/discovery_cache/file_cache.py", line 32, in <module>                                                                                                                                              
    from oauth2client.locked_file import LockedFile                                                                                                                                                                                                              
ImportError: No module named 'oauth2client.locked_file'   
```

It looks like oauth2client released v2.0.0 a few days ago and moved a bunch of stuff into a contrib module. At first glance it looks like a quick fix to be compatible with oauth2client v2.0.0. I'm willing to create a pull request if it helps.

**Comment by nathanielmanistaatgoogle** [comment]
Yes, a pull request would be very welcome!

**Comment by adrian-the-git** [comment]
Done did it already.

**Comment by adrian-the-git** [comment]
Think I fixed all the sample code; made pull request.

**Comment by nathanielmanistaatgoogle** [comment]
This was fixed in #182.
